src/batterymonitor.h

```diff
@@ -54,6 +54,10 @@
     #define BATTERY_SHIELD_R2 220.0
 #endif
 
+#ifndef BATTERY_REGULATOR
+    #define BATTERY_REGULATOR REG_LDO
```

Comment:
To keep backwards compatibility the fallback should be legacy behavior imo.
```suggestion
    #define BATTERY_REGULATOR REG_LEGACY
```

Reply:
I mean fair, but then why even keep it around in the first place. At the very least it should be official slime option as default then imo. I'll be setting up a test today.

Reply:
official slime uses a buck regulator, which will draw roughly constant power, the discharge characteristic is different than most diy boards which have a linear ldo regulator, which will draw roughly constant current